### PR TITLE
Add tests to controller when TLS is enabled

### PR DESCRIPTION
**Pull request by rubenvp8510**

### CODE REVIEW

File: controllers/tempo/suite_test.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ func TestMain(m *testing.M) {
 		//	Paths: []string{filepath.Join("..", "config", "webhook")},
 		//},
 	}
+	testEnv.BinaryAssetsDirectory = "/home/rvargasp/.local/share/kubebuilder-envtest/k8s/1.24.2-linux-amd64/"
```
Comment (iblancasa) [inline controllers/tempo/suite_test.go:29]: This should not be needed since `KUBEBUILDER_ASSETS` is set.

**Review by iblancasa — CHANGES_REQUESTED** [review]
LGTM. Just a minor change

File: controllers/tempo/suite_test.go (hunk 1 above)
Comment (rubenvp8510) [reply controllers/tempo/suite_test.go]: Done, thanks!

Comment (andreasgerstmayr) [reply controllers/tempo/suite_test.go]: It's required when you run the test from the "Testing" tab in VS Code. I'd also like this fixed somehow, but of course we don't want to hardcode this absolute path (including a version) :|

Comment (rubenvp8510) [reply controllers/tempo/suite_test.go]: Yes  exactly, I tried to put in my `launch.json` the env variable `KUBEBUILDER_ASSETS` and it didn't work, so this was an easy "fix". :D

**Review by iblancasa — APPROVED** [review]

**Pull request merged**